xmodule/item_bank_block.py

```diff
@@ -475,7 +478,7 @@ def validate(self):
                     action_label=_("Edit the problem bank configuration."),
                 )
             )
-        elif len(self.children) < self.max_count:
+        elif len(self.children) > 0 and len(self.children) < self.max_count:
```

Comment:
[nit, optional]: You can also do this as `elif 0 < len(self.children) < self.max_count`